holmes/core/investigation_structured_output.py

```diff
@@ -1,22 +1,29 @@
-from typing import Any, Dict, Optional, Tuple, Union
+import logging
+from typing import Any, Dict, Optional, Tuple
 import json
+import re
+import os
 
-from pydantic import RootModel
 
-InputSectionsDataType = Dict[str, str]
+REQUEST_STRUCTURED_OUTPUT_FROM_LLM = os.environ.get(
```

Comment:
we have a method for loading booleans: `load_bool` 
might be better to use it, to be consistent